components/style/properties/longhand/box.mako.rs

```diff
@@ -129,12 +133,14 @@ ${helpers.single_keyword("position", "static absolute relative fixed",
 
 ${helpers.single_keyword("clear", "none left right both",
                          animatable=False, gecko_ffi_name="mBreakType",
-                         gecko_enum_prefix="StyleClear")}
+                         gecko_enum_prefix="StyleClear",
+                         spec="https://www.w3.org/TR/CSS2/visuren.html#flow-control")}
 
 <%helpers:longhand name="-servo-display-for-hypothetical-box"
                    animatable="False"
                    derived_from="display"
-                   products="servo">
+                   products="servo"
+                   spec="Nonstandard (for user-agent stylesheets only)">
```

Comment:
Instead of this, I'd probably write "Internal (not web-exposed)", what do you think?